src/app/components/Recommendations/RecommendationsItem/index.tsx

```diff
@@ -25,9 +25,11 @@ const RecommendationsItem = ({
 
   const { title, image, href } = recommendation;
 
+  const hasImage = image && image.locator && image.originCode && image.altText;
+
   return (
     <div css={styles.promoWrapper} data-e2e="recommendations-wrapper">
-      {!isLite && (
+      {hasImage && !isLite && (
```

Comment:
Can we move this logic to within the RecommendationsImage component? If there's no image (and the other fields required for rendering, then return null?)